tests/analyzers_manager/__init__.py

```diff
@@ -94,6 +94,10 @@ class _ObservableAnalyzersScriptsTestCase(_AbstractAnalyzersScriptTestCase):
             "max_tries": 1,
             "poll_distance": 1,
         },
+        "HaveIBeenPwned": {
+            "max_tries": 1,
+            "domain": "",
```

Comment:
I fixed this but the test is failing due to black formatter. I think It looks like the error is caused by a trailing comma in the `api_app/core/dataclasses.py` file on the line that says `immutable=True,` but i am not able to find this parameter in the file.

Reply:
I fixed this! but the linters test are failing as It looks like the error is caused by a trailing comma in the `api_app/core/dataclasses.py` file on the line that says `immutable=True,`.
However i am not able to find the immutable parameter in the file.

Reply:
Also my bad! I was at first thinking it could be due to the changes in my file! That's why so many reverts and commits and the commit history got messy. I'll just stash them into one once the tests pass.

Reply:
don't worry about making the history bad here because, once merging, we will always force squash the commits